src/runtime/auto-prompt-manager.ts

```diff
@@ -704,8 +716,11 @@ export class AutoPromptManager {
       return actions[0];
     }
 
-    const globalFrequencyCapDuration =
-      frequencyCapConfig?.globalFrequencyCap?.frequencyCapDuration;
+    const isSubscriptionOpenaccess =
+      this.isSubscription_(autoPromptType) && !this.pageConfig_.isLocked();
+    const globalFrequencyCapDuration = isSubscriptionOpenaccess
+      ? SUBSCRIPTION_OPENACCESS_GLOBAL_FREQUENCY_CAP_DURATION
+      : frequencyCapConfig?.globalFrequencyCap?.frequencyCapDuration;
```

Comment:
Can we set up default value for global frequency in the server? Then I guess subscription & openaccess would run through the rest of logic.

Reply:
Will move this to server side.